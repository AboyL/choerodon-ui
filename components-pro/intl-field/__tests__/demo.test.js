import demoTest from '../../../tests/shared/demoTest';

demoTest('intl-field-pro');
