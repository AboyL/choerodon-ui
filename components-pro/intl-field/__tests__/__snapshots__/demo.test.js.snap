// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components-pro/intl-field/demo/dataset.md correctly 1`] = `
<div
  class="c7n-row"
  style="margin-left:-5px;margin-right:-5px"
>
  <div
    class="c7n-col-12"
    style="padding-left:5px;padding-right:5px"
  >
    <span
      class="c7n-pro-input-wrapper c7n-pro-input-required c7n-pro-input-suffix-button"
    >
      <label>
        <input
          autocomplete="off"
          class="c7n-pro-input"
          name="first-name"
          type="text"
          value="吴"
        />
        <div
          class="c7n-pro-input-suffix"
        >
          <i
            class="icon icon-language"
          />
        </div>
      </label>
    </span>
    <span />
  </div>
  <div
    class="c7n-col-12"
    style="padding-left:5px;padding-right:5px"
  >
    <span
      class="c7n-pro-input-wrapper c7n-pro-input-required c7n-pro-input-empty c7n-pro-input-suffix-button"
    >
      <label>
        <input
          autocomplete="off"
          class="c7n-pro-input"
          name="first-name"
          placeholder="默认英文"
          type="text"
          value=""
        />
        <div
          class="c7n-pro-input-suffix"
        >
          <i
            class="icon icon-language"
          />
        </div>
      </label>
    </span>
    <span />
  </div>
</div>
`;
